src/language/walker/ruleWalker.ts

```diff
@@ -68,21 +64,14 @@ export class RuleWalker extends SyntaxWalker {
         }
     }
 
-    public skip(node: ts.Node) {
```

Comment:
I readded the method.
I will remove this in the followup PR with breaking changes